crates/router/src/connector/stripe/transformers.rs

```diff
@@ -157,6 +157,27 @@ pub struct PaymentIntentRequest {
     pub expand: Option<ExpandableObjects>,
     #[serde(flatten)]
     pub browser_info: Option<StripeBrowserInformation>,
+    #[serde(flatten)]
+    pub charges: Option<IntentCharges>,
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+#[serde(untagged)]
+pub enum IntentCharges {
+    Direct(DirectCharges),
+    Destination(DestinationCharges),
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct DirectCharges {
+    pub application_fee_amount: i64,
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct DestinationCharges {
+    pub application_fee_amount: i64,
+    #[serde(rename = "transfer_data[destination]")]
+    pub destination_account_id: String,
```

Comment:
Put the fields `application_fee_amount` and `destination_account_id`  in the `PaymentIntentRequest`